apps/web/src/contexts/globalLayout/GlobalNavbar/CollectionNavbar/CollectionNavbar.tsx

```diff
@@ -157,3 +155,10 @@ const MobileUsernameText = styled(Paragraph)`
 
   color: ${colors.shadow};
 `;
+
+const SlashText = styled(Paragraph)`
```

Comment:
moved `SlashText` from another file I deleted, bc this is the only place it's used now